src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -4,26 +4,48 @@
 
 import java.util.Objects;
 
+import seedu.address.model.entity.PrefixType;
+
 /**
  * Represents the result of a command execution.
  */
 public class CommandResult {
 
     private final String feedbackToUser;
 
-    /** Help information should be shown to the user. */
+    /**
+     * Help information should be shown to the user.
```

Comment:
Hey. I'm not sure whether there is a style/standard for this, but I was under the impression that for individual attributes, it's usually the // single line comment. Was there some kind of rule set for this? If there is, could you refer me to it, then i can edit the comments and javadocs for my section of the code :)

Reply:
I'll ask during tomorrow's tutorial, but this is given under the orignal AddressBook 3 code haha